packages/core/src/Components/ECMASandbox.class.ts

```
@@ -0,0 +1,71 @@
+
+        const logger = this.createComponentLogger(agent, config);
+        try {
+            let Output: any = {};
+            let _error = undefined;
+
+            let codeInputs = {};
+            for (let fieldName in input) {
+                const _type = typeof input[fieldName];
+                switch (_type) {
+                    case 'string':
+                        const b64encoded = Buffer.from(input[fieldName]).toString('base64');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)